.github/workflows/application-onboarding-helm-template.yml

```diff
@@ -73,6 +76,10 @@ jobs:
         if: steps.directory_check.outputs.directory_exists == 'false'
         run: cp vsp-infra-helm/hello-world/Chart.yaml vsp-infra-application-manifests/apps/${{ inputs.application_name }}/${{ inputs.application_environment }}/Chart.yaml
 
+      - name: Modify Chart.yaml to have correct format
```

Comment:
What's the correct format?

Reply:
Here is an [example](https://github.com/department-of-veterans-affairs/vsp-infra-application-manifests/blob/main/apps/datadog-agent/dev/values.yaml) of correct format. Needs app-name then information